src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationSetting.cs

```diff
@@ -9,13 +9,22 @@
 
 namespace Azure.ApplicationModel.Configuration
 {
+    /// <summary>
+    /// ConfigurationSetting is a resource identified by unique combination of key and label.
```

Comment:
To me, the phrasing is a bit awkward.   Perhaps we should consider something like `A setting, defined by a unique combination of a key and label`.